charts/openshift-reporting/templates/report-queries/persistentvolumeclaim-usage.yaml

```diff
@@ -175,4 +175,50 @@ spec:
     AND "timestamp" < timestamp '{| default .Report.ReportingEnd .Report.Inputs.ReportingEnd | prestoTimestamp |}'
     AND dt >= '{| default .Report.ReportingStart .Report.Inputs.ReportingStart | prometheusMetricPartitionFormat |}'
     AND dt <= '{| default .Report.ReportingEnd .Report.Inputs.ReportingEnd | prometheusMetricPartitionFormat |}'
-    GROUP BY namespace, persistentvolumeclaim
+
+---
+
+apiVersion: metering.openshift.io/v1alpha1
+kind: ReportGenerationQuery
+metadata:
+  name: "namespace-persistentvolumeclaim-usage"
+  labels:
+    operator-metering: "true"
+{{- block "extraMetadata" . }}
+{{- end }}
+spec:
+  reportQueries:
+  - "persistentvolumeclaim-usage-with-phase-raw"
+  view:
+    disabled: true
+  columns:
+  - name: period_start
+    type: timestamp
+    unit: date
+  - name: period_end
+    type: timestamp
+  - name: namespace
+    type: string
+    unit: kubernetes_namespace
+  - name: persistentvolumeclaim
+    type: string
+    unit: kubernetes_persistentvolumeclaim
+  - name: persistentvolumeclaim_usage_bytes
+    type: double
+    unit: bytes
+  inputs:
+  - name: ReportingStart
+  - name: ReportingEnd
+  query: |
+    SELECT
+      timestamp '{| default .Report.ReportingStart .Report.Inputs.ReportingStart| prestoTimestamp |}' AS period_start,
+      timestamp '{| default .Report.ReportingEnd .Report.Inputs.ReportingEnd | prestoTimestamp |}' AS period_end,
+      namespace,
+      persistentvolumeclaim,
```

Comment:
This cannot be a column since it's not aggregated nor in the group by. Remove it

Reply:
Fixed in commit c08577fb4129f09b07338ad2b00134157360c62f